packages/atomic/stencil.config.ts

```diff
@@ -17,6 +17,7 @@ const isDevWatch: boolean =
 export const config: Config = {
   namespace: 'atomic',
   taskQueue: 'async',
+  globalStyle: 'src/styles/variables.pcss',
```

Comment:
eventually we could have multiple global variable files, for different themes that we would want to provide

Reply:
Do we want to use `globalStyle` here, it builds one unified `dist/atomic.css` file, but if we intend for there to be more themes we probably want separately defined files. Since we're not using any of the PostCSS features inside `variables.pcss` I think we could have a top level themes folder (outside `src`) with this as regular css file`default.css` and then import that inside `index.html`.